Sources/KituraNet/HTTP/HTTPServer.swift

```diff
@@ -309,15 +324,16 @@ public class HTTPServer: Server {
             }
             .childChannelInitializer { channel in
                 let httpHandler = HTTPRequestHandler(for: self)
-                let config: NIOHTTPServerUpgradeConfiguration = (upgraders: upgraders, completionHandler: { _ in
+                let config: HTTPUpgradeConfiguration = (upgraders: upgraders, completionHandler: { ctx in
+                    self.ctx = ctx
```

Comment:
Does this new property `ctx` ever get read? I can't see reference to it elsewhere in this PR